en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,186 @@
+Medias display
+==============
+
+Context
+-------
```

Comment:
I am not convinced.

Moreover a say that we can install the media-bundle as a standalone application : https://github.com/open-orchestra/open-orchestra-docs/pull/81/files#diff-4c0f5d5e8374269fce28c2f5d65581e5R28